code/game/machinery/iv_drip.dm

```diff
@@ -3,6 +3,7 @@
 
 /obj/machinery/iv_drip
 	name = "\improper IV drip"
+	desc = "Simply attach a bloodbag, and stab the patient, and they'll have more blood in no time."
```

Comment:
```suggestion
	desc = "Simply attach a blood bag, and stab the patient, and they'll have more blood in no time. You can put other stuff in the bags too, I guess."
```

Reply:
I feel like thats something for the blood bag itself